frontend21/scss/amp-dev.scss

```diff
@@ -70,15 +71,18 @@ h5,
 h6 {
   margin-top: 0;
   margin-bottom: 0.3em;
+  color: currentColor;
```

Comment:
why is this being done? [per spec](https://drafts.csswg.org/css-color/#currentcolor-color)

```
If the ‘currentColor’ keyword is set on the ‘color’ property itself, it is treated as ‘color: inherit’.
```

Its technically not invalid, but just sorta confusing. Are you wanting to inherit the value but not cascade it?

Reply:
True, it makes sense to use ```color: inherit;``` then, unless @matthiasrohmer had some reason to implement it that way.